{
    "config": {
        "error": {
            "cannot_connect": "Erreur de connexion",
            "invalid_auth": "Activer l'authentification dans Pr\u00e9f\u00e9rences > Authentification de m\u00fctesync",
            "unknown": "Erreur inattendue"
        },
        "step": {
            "user": {
                "data": {
                    "host": "H\u00f4te"
                }
            }
        }
    }
}